README.md

```diff
@@ -16,14 +16,17 @@ Feel free to use them anyway, if you wish!
 
 The codes use a multitude of packages as can be found in the [`setup.py`](setup.py) file.
 
-Important ones are: `numpy`, `pandas` and `scipy`.
+Important ones are: `numpy`, `pandas`, `scipy`, `tfs-pandas`, `sdds` and `generic_parser`.
```

Comment:
I would mention in some way that tfs-pandas, sdds and generic-parser are/were parts of this repo, our codes... Currently it sounds like we use random unknown packages.